gulpfile.js

```diff
@@ -0,0 +1,114 @@
+const gulp = require('gulp');
+
+const { series, parallel } = require('gulp');
+const del = require('del');
+const concat = require('gulp-concat');
+const uglify = require('gulp-uglify');
+const merge = require('merge-stream');
+const replace = require('gulp-replace');
+const cleancss = require('gulp-clean-css');
+
+const plugin = {
+  'public': './webroot'
+}
+
+const main_styles = [
+  './node_modules/leaflet/dist/leaflet.css',
+  './node_modules/leaflet.fullscreen/Control.FullScreen.css',
+  './node_modules/leaflet-gesture-handling/dist/leaflet-gesture-handling.css',
+  './node_modules/leaflet.markercluster/dist/MarkerCluster.css',
+  './node_modules/leaflet.markercluster/dist/MarkerCluster.Default.css',
+  './node_modules/leaflet-control-geocoder/dist/Control.Geocoder.css',
+  './node_modules/leaflet-legend/leaflet-legend.css'
+];
+
+const main_scripts = [
+  './node_modules/leaflet/dist/leaflet.js',
+  './node_modules/leaflet.fullscreen/Control.FullScreen.js',
+  './node_modules/leaflet-gesture-handling/dist/leaflet-gesture-handling.min.js',
+  './node_modules/leaflet.markercluster/dist/leaflet.markercluster.js',
+  './node_modules/leaflet-control-geocoder/dist/Control.Geocoder.js',
+  './node_modules/leaflet-legend/leaflet-legend.js'
+];
+
+const main_assets = [
+  {
+    'src': './node_modules/leaflet/dist/images/*',
+    'dest': '/images/'
+  },
+  {
+    'src': './node_modules/leaflet.locatecontrol/*.svg',
```

Comment:
Although svg files are stored in the plugin's images folder and their path later replaced in the css files with the new one, trying to reach them throws an error 😕

```
Type : Error
Code : 0
Message : Call to undefined method DI\Container::halt()
Fichier : [...]/galette/galette/includes/routes/plugins.routes.php
Ligne : 71 
```

Reply:
This is an error in Galette core; the `svg` extension is not known. To make this work, you just need to add the extension in the `$auth_ext` array (`galette/includes/routes/plugins.routes.php`).
I'll have to take a look for the `halt` replacement, as far as I remmeber, this method is no longer part of Slim for a while.

Reply:
Simple as that !
I should have followed the line 😅